hazelcast/src/test/java/com/hazelcast/client/protocol/compatibility/ReferenceObjects.java

```diff
@@ -716,7 +717,8 @@ private static boolean isEqualStackTrace(StackTraceElement stackTraceElement1, S
 
     public static ListenerConfigHolder aListenerConfigHolder = new ListenerConfigHolder(anInt, aData, aString, aBoolean, aBoolean);
     public static AttributeConfig anAttributeConfig = new AttributeConfig(aString, aString);
-    public static IndexConfig anIndexConfig = CustomTypeFactory.createIndexConfig(aString, anEnum, aListOfStrings);
+    public static BitmapIndexOptions aBitmapIndexOptions = new BitmapIndexOptions();
```

Comment:
this should have default parameters set. aString for unique key and anInt for uniqueKeyTransformation fields